settings.php

```diff
@@ -72,6 +72,13 @@
             1)
     );
 
+    // Include libraries in backup.
+    $settings->add(
+        new admin_setting_configcheckbox('mod_hvp_backup_libraries',
+        get_string('backuplibraries', 'hvp'),
+        get_string('backuplibraries_help', 'hvp'), 1)
```

Comment:
Indentation levels are a bit off here, e.g. I see the 1 is the default value but it's on the same line as the help string.

Reply:
Matching format around it. I can change it.